test/mciproject.yml

```diff
@@ -21,25 +33,53 @@ post:
         rm -rf "wiredtiger"
 
 tasks:
+## Base compile task on posix flavours
   - name: compile-posix
+    depends_on: []
     commands:
       - func: "fetch source"
       - command: git.apply_patch
         params:
           directory: wiredtiger
       - command: shell.exec
         params:
-          working_dir: "wiredtiger"
+          working_dir: "wiredtiger/build_posix"
           script: |
             set -o errexit
             set -o verbose
 
-            ./build_posix/reconf
-            ${configure_env_vars|} ./configure --enable-diagnostic --enable-python --enable-zlib
+            ./reconf
+            ${configure_env_vars|} ../configure --enable-diagnostic --enable-python --enable-zlib
             ${make_command|make} ${smp_command|}
             ${make_command|make} check
+      - command: shell.exec
```

Comment:
use `archive.targz_pack` instead, see https://github.com/mongodb/mongo/blob/master/etc/evergreen.yml for examples.